low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -47,5 +47,85 @@
                     "Disk in slot '{}' has recovered. [{}]",
                     "Disk is in good health now.", "None")
         })
+    },
+    "Power Supply": {
+        **dict.fromkeys(["Presence detected", "Presence detected ()"], {
+            "Asserted": Alert(
+                "insertion", "informational",
+                "'Power Supply {}' is present. [{}]",
+                # Intel/Supermicro: 'Power Supply 1' is present. [PS1 (0x85)]
+                "Power Supply Port is being monitored.", "None"),
+            "Deasserted": Alert(
+                "missing", "critical",
+                "'Power Supply {}', is missing. [{}]",
+                "Power Supply Port monitoring has stopped.",
```

Comment:
Impact - Power supply redundancy is affected, if more power supplies also goes down or missing, server may go offline.
Recommendation - Install missing Power supply immediately

Reply:
changed. 👍